pkg/vpcmodel/jsonOutput.go

```diff
@@ -38,6 +46,14 @@ type connLine struct {
 	UnidirectionalConn common.ConnDetails `json:"unidirectional_conn,omitempty"`
 }
 
+type diffLine struct {
+	Diff  string             `json:"diff-type"`
+	Src   EndpointElem       `json:"src"`
+	Dst   EndpointElem       `json:"dst"`
+	Conn1 common.ConnDetails `json:"conn1"`
+	Conn2 common.ConnDetails `json:"conn2"`
+}
+
```

Comment:
missing `diff-info` in case of added/removed subnet/vsi?

Reply:
No. For json mode the exact diff type is described - e.g. "source removed" or "source and destination added" or "connection removed". I figured that since this is machine readable, having the info more concise makes more sense.